### DISCUSSION TITLE
OTA for secure application NCP

### DISCUSSION THREAD

**Discussion by hassan-xy** [zwave-js/zwave-js-ui#3906] [Q&A]
Hi team,
I need to know if its possible to perform signed OTA update for the NCP firmware for zwave 700 series ZGM130S037HGN if the device is configured with secure application boot.

Please share the procedure if its applicable.

Thanks

**Comment by robertsLando** [comment]
cc @AlCalzone

**Comment by AlCalzone** [comment]
That's a question for [Silicon Labs](https://community.silabs.com/s/topic/0TO1M000000qHcQWAU/zwave?language=en_US).
From the Z-Wave network perspective, OTA updates are done via the `Firmware Update Meta Data CC`, no matter the end device. See https://zwave-js.github.io/node-zwave-js/#/api/node?id=updatefirmware

**Comment by hassan-xy** [comment]
Thanks for the reply @AlCalzone. Will your interface work for signed NCP files also?

**Reply by AlCalzone** [reply]
The contents of the OTA update files are mostly opaque to Z-Wave JS.
HEX-encoded files (usually end in .hex, .otz, .ota) and some other proprietary formats are converted to binary first.
.gbl and other binary files are transferred as-is.
